2023-11-25 05:58:49,348 INFO    MainThread:3885 [wandb_init.py:_log_setup():292] Logging user logs to /content/drive/MyDrive/on-policy/onpolicy/scripts/results/StarCraft2/2m_vs_1z/rmappo/check/wandb/run-20231125_055848-fdvz7suj/logs/debug.log
2023-11-25 05:58:49,350 INFO    MainThread:3885 [wandb_init.py:_log_setup():293] Logging internal logs to /content/drive/MyDrive/on-policy/onpolicy/scripts/results/StarCraft2/2m_vs_1z/rmappo/check/wandb/run-20231125_055848-fdvz7suj/logs/debug-internal.log
2023-11-25 05:58:49,351 INFO    MainThread:3885 [wandb_setup.py:_flush():68] setting env: {}
2023-11-25 05:58:49,352 INFO    MainThread:3885 [wandb_setup.py:_flush():68] setting user settings: {}
2023-11-25 05:58:49,352 INFO    MainThread:3885 [wandb_setup.py:_flush():68] multiprocessing start_methods=fork,spawn,forkserver
2023-11-25 05:58:54,181 INFO    MainThread:3885 [wandb_run.py:_console_start():1265] atexit reg
2023-11-25 05:58:54,188 INFO    MainThread:3885 [wandb_run.py:_redirect():1135] redirect: SettingsConsole.REDIRECT
2023-11-25 05:58:54,189 INFO    MainThread:3885 [wandb_run.py:_redirect():1138] Redirecting console.
2023-11-25 05:58:54,190 INFO    MainThread:3885 [redirect.py:install():196] install start
2023-11-25 05:58:54,190 INFO    MainThread:3885 [redirect.py:install():211] install stop
2023-11-25 05:58:54,191 INFO    MainThread:3885 [redirect.py:install():196] install start
2023-11-25 05:58:54,201 INFO    MainThread:3885 [redirect.py:install():211] install stop
2023-11-25 05:58:54,202 INFO    MainThread:3885 [wandb_run.py:_redirect():1182] Redirects installed.
2023-11-25 05:59:00,872 INFO    Dummy-4   :3885 [wandb_run.py:_atexit_cleanup():1238] got exitcode: 1
2023-11-25 05:59:00,874 INFO    Dummy-4   :3885 [wandb_run.py:_restore():1210] restore
2023-11-25 05:59:00,875 INFO    Dummy-4   :3885 [redirect.py:uninstall():215] uninstall start
2023-11-25 05:59:00,875 INFO    Dummy-4   :3885 [redirect.py:_stop():264] _stop: stdout
2023-11-25 05:59:00,876 INFO    stdout    :3885 [redirect.py:_pipe_relay():119] relay done saw last write: stdout
2023-11-25 05:59:00,876 INFO    stdout    :3885 [redirect.py:_pipe_relay():135] relay done done: stdout
2023-11-25 05:59:00,876 INFO    Dummy-4   :3885 [redirect.py:_stop():270] _stop closed: stdout
2023-11-25 05:59:00,877 INFO    Dummy-4   :3885 [redirect.py:_stop():276] _stop joined: stdout
2023-11-25 05:59:00,877 INFO    Dummy-4   :3885 [redirect.py:_stop():278] _stop rd closed: stdout
2023-11-25 05:59:00,877 INFO    Dummy-4   :3885 [redirect.py:uninstall():219] uninstall done
2023-11-25 05:59:00,877 INFO    Dummy-4   :3885 [redirect.py:uninstall():215] uninstall start
2023-11-25 05:59:00,877 INFO    Dummy-4   :3885 [redirect.py:_stop():264] _stop: stderr
2023-11-25 05:59:00,877 INFO    stderr    :3885 [redirect.py:_pipe_relay():119] relay done saw last write: stderr
2023-11-25 05:59:00,877 INFO    stderr    :3885 [redirect.py:_pipe_relay():135] relay done done: stderr
2023-11-25 05:59:00,878 INFO    Dummy-4   :3885 [redirect.py:_stop():270] _stop closed: stderr
2023-11-25 05:59:00,878 INFO    Dummy-4   :3885 [redirect.py:_stop():276] _stop joined: stderr
2023-11-25 05:59:00,878 INFO    Dummy-4   :3885 [redirect.py:_stop():278] _stop rd closed: stderr
2023-11-25 05:59:00,878 INFO    Dummy-4   :3885 [redirect.py:uninstall():219] uninstall done
2023-11-25 05:59:00,885 INFO    Dummy-4   :3885 [wandb_run.py:_wait_for_finish():1349] got exit ret: uuid: "3066e8c2c9b7485584cb844910563b55"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 1268
      total_bytes: 1268
    }
  }
}

2023-11-25 05:59:02,891 INFO    Dummy-4   :3885 [wandb_run.py:_wait_for_finish():1349] got exit ret: uuid: "61775da115954b769b68ab2b09f60028"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 8083
      total_bytes: 8083
    }
  }
}

2023-11-25 05:59:04,032 INFO    Dummy-4   :3885 [wandb_run.py:_show_files():1543] logging synced files
2023-11-25 05:59:04,034 INFO    Dummy-4   :3885 [internal.py:handle_exit():137] Internal process exited
